### PR TITLE
Dbld: Update package list for Debian Jessie

### PR DESCRIPTION
**Pull request by mitzkia**

### CODE REVIEW

File: dbld/images/required-apt/all.txt [hunk 1]
```diff
@@ -44,10 +44,12 @@ strace
 vim
 
 # debian packaging tools
+fakeroot
```
Comment (szemere) [inline dbld/images/required-apt/all.txt]: If you put a package into the "all.txt" list, than you should:
- check if the package is really available on all platforms (fakeroot is)
- remove it from the distribution specific lists (in this case: trusty)

Comment (mitzkia) [reply dbld/images/required-apt/all.txt]: Thanks for the feedback.
I have removed dh-exec and fakeroot from platform specific files.

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Pull request merged**